NEWS.md

```diff
@@ -1,3 +1,8 @@
+# DeeDeeExperiment 0.99.1
+
+* `DeeDeeExperiment` now extends the `SingleCellExperiment` class, accommodating
+both bulk and single-cell analysis workflows
+
```

Comment:
See note above, but I think it is a fine thing to do to call it a different number